vllm/worker/neuron_model_runner.py

```diff
@@ -75,6 +76,15 @@ def __init__(
 
         # Lazy initialization.
         self.model: nn.Module  # initialize after load_model.
+        self.model_config.generation_config = GenerationConfig(
+            max_length=self.scheduler_config.max_model_len,
+            do_sample=True,
+            per_batch_line=True,
+            top_k=[1] * self.scheduler_config.max_num_seqs,
+            top_p=[1] * self.scheduler_config.max_num_seqs,
```

Comment:
The default for sampling_params.top_p is 1.0 (floating-point) instead 1.

Reply:
ack.